scripts/generate_testscases.py

```diff
@@ -120,15 +122,25 @@ def function_name():
                 )
 
             for i, curr_output in enumerate(outputs):
+                if not isinstance(curr_output, np.ndarray):
+                    if isinstance(curr_output, int):
```

Comment:
We should check if tf.int_64 is also represented as Python int here. It's worth adding an e2e test for both cases in a follow up.